### ISSUE TITLE
Not working on my app

### ISSUE THREAD

**Issue by waldo1001** [martonsagi/al-object-designer#18]
Hi, I'm testing this extension - looks awesome!  You even not invented warm water, and included my extension to run objects, thanks :-).

I have a problem when executing this on my "MostUselessAppEver" though:
https://github.com/waldo1001/MostUselessAppEver

May be you can clone this one, and try it out to find out what's wrong?

It seems to keep discovering ...
![image](https://user-images.githubusercontent.com/12088142/51736145-a1e10d80-2089-11e9-9bbc-70f5aa3e21ea.png)

**Comment by martonsagi** [comment]
Hi Waldo, thanks! :)

[This issue](https://github.com/martonsagi/al-object-designer/issues/15) has been solved in [0.0.6-alpha.4](https://github.com/martonsagi/al-object-designer/blob/0.0.6-dev/custom-builds/al-object-designer-0.0.6-alpha.4.vsix). I'm planning to publish the final v0.0.6 in the weekend.

I've just tested 0.0.6-alpha.4 with MostUselessAppEver and Object Designer worked for me. On the other hand though, your app breaks Page Designer, so it may be useful for edge cases after all. :)
